resources/views/community/components/event/aotw.blade.php

```diff
@@ -1,31 +1,49 @@
+@props([
+    'achievement' => [],
+    'game' => [],
+    'consoleName' => '',
+    'forumTopicId' => 0,
+])
+
 <?php
+$achievementId = $achievement->ID;
+$achievementName = $achievement->Title;
+$achievementDescription = $achievement->Description;
+$achievementPoints = $achievement->Points;
+$achievementRetroPoints = $achievement->TrueRatio;
+$achievementBadgeName = $achievement->BadgeName;
 
-use App\Site\Models\StaticData;
+$renderedGameTitle = renderGameTitle($game->Title);
+$achievementIconSrc = media_asset("/Badge/$achievementBadgeName.png");
+$gameSystemIconUrl = getSystemIconUrl($game->ConsoleID);
+?>
 
-$staticData = StaticData::first();
+<div class="component">
+    <h3>Achievement of the Week</h3>
 
-if ($staticData === null) {
-    return;
-}
+    <div class="bg-embed p-4 rounded border border-embed-highlight">
+        <a href="{{ route('achievement.show', $achievement->ID) }}" class="group flex gap-x-2 mb-3 text-text transition-transform hover:-translate-y-0.5">
```

Comment:
I don't like things moving when I hover over them. I assumed this was a bug until I saw the code intentionally doing it.

https://github.com/RetroAchievements/RAWeb/assets/32680403/88c72ff7-7bb6-4a56-b8d5-25fa5a1f07a1

Reply:
No worries at all, this should now be addressed.